### PR TITLE
quarantine_zephyr: Add zephyr tests to overflow issues

### PR DESCRIPTION
**Pull request by katgiadla**
Add Zephyr tests due to overflow issues.

Refers:
NCSDK-31589
NCSDK-31590
NCSDK-31591

### CODE REVIEW

File: scripts/quarantine_zephyr.yaml [hunk 1]
```diff
@@ -216,6 +216,28 @@
...
+    - drivers.eeprom.emul.build
+  platforms:
+    - nrf54h20dk@0.9.0/nrf54h20/cpuppr
+  comment: "https://nordicsemi.atlassian.net/browse/NCSDK-31589"
+
+- scenarios:
+    - llext.writable
+    - llext.writable_relocatable
+    - llext.writable_slid_linking
+    - llext.writable_relocatable_slid_linking
```
Comment (PerMac) [inline scripts/quarantine_zephyr.yaml:229]: ~~there is already an entry forr llext above, can you pleas just add this platform there?~~ seems to be a different issue

**Comment by anangl** [comment]
Cherry-picked into the branch.
